lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
> add a `scorePrecision` or similarly named property to the performance audits meta object specifying a numeric precision

should be noted that is basically just giving `expectedValue` a new name :)

It will be nice to always have a numeric score, though.